server/schema/admin/res.js

```diff
@@ -0,0 +1,30 @@
+import {Admin} from '../../models'
+import bcrypt from 'bcrypt'
+import jwt from 'jsonwebtoken'
+const resolvers = {
+  RootQuery: {},
+  RootMutation: {
+    async loginAdmin(_, {input}) {
+      const admin = await Admin.findOne({where: {username: input.username}})
+      if (!admin) {
+        throw new Error('Account not found. Please try again!')
+      }
+      const valid = await bcrypt.compare(input.password, admin.password)
+      if (!valid) {
+        throw new Error('Wrong Password ...')
+      }
+      return (
```

Comment:
abundant  "(" ?

Reply:
eslint auto for like that...can't remove it